types/ext/dictionary-database.d.ts

```diff
@@ -118,12 +118,19 @@ export type DatabaseTermMetaPitch = {
     dictionary: string;
 };
 
+export type DatabaseTermMetaIpa = {
```

Comment:
Naming suggestion: Instead of using `ipa` and variants in type names, use `PhoneticTranscription`. The `'ipa'` is good for the dictionary data schema, since shortening there is better for overall file/database size.